lesson37/src/js/password.js

```diff
@@ -0,0 +1,122 @@
+import { Chance } from "chance";
+const passwordButtons = [...document.querySelectorAll(".js-password-icon")];
```

Comment:
I learned a lot about how to write an array with spread syntax.
Because it will be replaced with an array
I didn't have the idea to enable array operations.